src/components/headline/index.tsx

```diff
@@ -21,6 +21,7 @@ const StyledHeadline = styled.div`
 	margin-top: 0;
 	font-family: ${fonts().NORMAL_FONT};
 	font-weight: 500;
+	user-select: none;
```

Comment:
I am not a huge fan of disableing text selection for entire typography styles. If we positively need to block it somewhere we should do it on that specific screen.